src/Microsoft.ML.Transforms/KeyToVectorMapping.cs

```diff
@@ -426,8 +426,20 @@ private void EncodeValueToBinary(BufferBuilder<float> bldr, uint value, int bits
     }
 
     /// <summary>
-    ///  Converts the key types back to binary vectors.
+    /// Estimator for <see cref="KeyToBinaryVectorMappingTransformer"/>. Converts key types to their corresponding binary representation.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | [key](xref:Microsoft.Ml.Data.KeyDataViewType) |
+    /// | Output column data type | A vector of [System.Single](xref:System.Single). |
```

Comment:
>[System.Single](xref:System.Single) [](start = 48, length = 35)

<xref:System.Single>

If the anchor text is the same as the class/type name, you don't really need to use [anchor](xref:UID) pattern. You can save some time by using autolinks: <xref:UID>